charts/argo-cd/templates/argocd-application-controller/statefulset.yaml

```diff
@@ -210,7 +210,7 @@ spec:
           {{- toYaml . | nindent 10 }}
         {{- end }}
         ports:
-        - name: controller
```

Comment:
There is no controller port - just metrics one. See https://github.com/argoproj/argo-cd/blob/05dce81d09afde3e53886f1acaf337bb203e18e3/cmd/argocd-application-controller/commands/argocd_application_controller.go#L188